File: soccer-league/soccer-league/Data/DTOs/PlayerDTO.swift
import Foundation

struct PlayerDTO: Codable {
    let id: Int
    let name: String?          // nickname / in-game name ("name" in API)
    let firstName: String?     // decoded from "first_name"
    let lastName: String?      // decoded from "last_name"
    let imageUrl: String?      // decoded from "image_url"
}

File: csgo-leagues-android/app/src/main/java/com/voidx/csgo/data/dto/PlayerDTO.kt
package com.voidx.csgo.data.dto

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class PlayerDTO(
    val id: Int,
    val name: String? = null,
    @SerialName("first_name") val firstName: String? = null,
    @SerialName("last_name") val lastName: String? = null,
    @SerialName("image_url") val imageUrl: String? = null,
)
